open-ce/env_config.py

```diff
@@ -49,14 +49,13 @@ def _validate_config_file(env_file, variants):
     try:
         meta_obj = conda_build.metadata.MetaData(env_file, variant=variants)
         if not (Key.packages.name in meta_obj.meta.keys() or Key.imported_envs.name in meta_obj.meta.keys()):
-            raise Exception("Content Error!",
-                            "An environment file needs to specify packages or "
-                            "import another environment file.")
+            raise OpenCEError("Content Error!:\n"
+                              "An environment file needs to specify packages or "
+                              "import another environment file.")
         utils.validate_dict_schema(meta_obj.meta, _ENV_CONFIG_SCHEMA)
         return meta_obj
     except (Exception, SystemExit) as exc: #pylint: disable=broad-except
-        print('***** Error in %s:\n  %s' % (env_file, exc), file=sys.stderr)
-        return None
+        raise OpenCEError('***** Error in {}:\n  {}'.format(env_file, str(exc))) from exc
```

Comment:
Just a nitpick - Adding asteriks to the error message helps it to be highlighted. So, probably it is good to have for all error messages.

Reply:
Yeah, it is very easy to differentiate Open-CE errors from others when these special delimiters.

Reply:
All errors will now start with `[OPEN-CE-ERROR-#]` as defined here: https://github.com/open-ce/open-ce/blob/76aed751074e17f6d1958758b31a2a7fa53408f6/open-ce/errors.py#L37